### PR TITLE
{phys}[intel/2016b] Elk v4.0.15

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/e/Elk/Elk-4.0.15-intel-2016.01.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+
+# make.inc file is generated interactively by "setup" command, creating it here
+prebuildopts = 'echo "F90_OPTS = $FFLAGS" > make.inc && '
+prebuildopts += 'echo "F77_OPTS = $FFLAGS" >> make.inc && '
+prebuildopts += 'echo "LIB_LPK = $LIBLAPACK" >> make.inc && '
+prebuildopts += 'echo "LIB_libxc = $EBROOTLIBXC/lib/libxcf90.a $EBROOTLIBXC/lib/libxc.a" >> make.inc && '
+prebuildopts += 'echo "SRC_libxc = libxc_funcs.f90 libxc.f90 libxcifc.f90" >> make.inc && '
+prebuildopts += 'echo "SRC_FFT = zfftifc_fftw.f90" >> make.inc && '
+
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/e/Elk/Elk-4.0.15-intel-2016.01.eb]: better way?

File: easybuild/easyconfigs/e/Elk/Elk-4.0.15-intel-2016.01.eb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+dependencies = [
+    ('libxc', '2.2.3'),
+]
+
+
+# make.inc file is generated interactively by "setup" command, creating it here
+prebuildopts = 'echo "F90_OPTS = $FFLAGS" > make.inc && '
+prebuildopts += 'echo "F77_OPTS = $FFLAGS" >> make.inc && '
+prebuildopts += 'echo "LIB_LPK = $LIBLAPACK" >> make.inc && '
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/e/Elk/Elk-4.0.15-intel-2016.01.eb]: better way?

File: easybuild/easyconfigs/e/Elk/Elk-4.0.15-intel-2016b.eb [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+dependencies = [
+    ('libxc', '2.2.3'),
+]
+
+
+# make.inc file is generated interactively by "setup" command, creating it here
+prebuildopts = 'echo "F90_OPTS = $FFLAGS" > make.inc && '
+prebuildopts += 'echo "F77_OPTS = $FFLAGS" >> make.inc && '
+prebuildopts += 'echo "LIB_LPK = $LIBLAPACK" >> make.inc && '
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/e/Elk/Elk-4.0.15-intel-2016b.eb]: better way?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2134.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/4bf144e2b973f00c0cce73362f4d1aac for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
node2301.phanpy.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/d36feb1570c449df20d738a7114fa787 for a full test report.

File: easybuild/easyconfigs/e/Elk/Elk-4.0.15-intel-2016.01.eb (hunk 2 above)
Comment (boegel) [inline easybuild/easyconfigs/e/Elk/Elk-4.0.15-intel-2016.01.eb]: @migueldiascosta let me know if you're happy with this as-is; with the successful test reports this is good to merge if you ask me

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @migueldiascosta!

**Pull request merged**